platform/jewel/scripts/compare-branches.main.kts

```diff
@@ -0,0 +1,222 @@
+        }
+
+        if (!branchExists(branch2, communityRoot)) {
+            printlnErr("Error: Branch '$branch2' not found.")
+            exitProcess(1)
+        }
+
+        // Ensure all paths to check exist
+        var anyPathMissing = false
+        for (path in Config.PATHS_TO_CHECK) {
+            val file = File(communityRoot, path)
+            if (!file.isDirectory()) {
+                printlnErr("Error: Path to check does not exist: $path")
+                anyPathMissing = true
+            }
+        }
+        if (anyPathMissing) exitProcess(1)
+
+        println(
+            "🔍 Comparing commits for paths '${Config.PATHS_TO_CHECK.joinToString()}' between '$branch1' and '$branch2' since $sinceDate..."
+        )
+
+        val pathsArg = Config.PATHS_TO_CHECK.joinToString(separator = " ") { "-- $it" }
+
+        val gitLogCommand1 = "git log $branch1 --after=$sinceDate --pretty=format:'${Config.GIT_LOG_FORMAT}' $pathsArg"
+        val commits1 =
+            runCommand(gitLogCommand1, communityRoot)
+                .output
+                .lines()
+                .filter { it.isNotBlank() }
+                .map { parseCommitLine(it) }
+
+        val gitLogCommand2 = "git log $branch2 --after=$sinceDate --pretty=format:'${Config.GIT_LOG_FORMAT}' $pathsArg"
+        val commits2 =
+            runCommand(gitLogCommand2, communityRoot)
+                .output
+                .lines()
+                .filter { it.isNotBlank() }
+                .map { parseCommitLine(it) }
+
+        if (verbose) {
+            if (commits1.isNotEmpty()) {
+                println("\n--- Commits for '$branch1' ---")
+                commits1.forEach(::printCommitInfoLine)
+                println()
+            }
+            if (commits2.isNotEmpty()) {
+                println("\n--- Commits for '$branch2' ---")
+                commits2.forEach(::printCommitInfoLine)
+                println()
+            }
+        }
+
+        val horizontalLine = "------------------------------------------------------------------"
+        println(horizontalLine)
+
+        val branch2Subjects = commits2.map { it.subject }.toSet()
+        val commitsInBranch1Only = commits1.filter { (_, subject) -> subject !in branch2Subjects }
+
+        if (verbose) {
+            println("Commits in '$branch1' but not '$branch2':")
+            println(commitsInBranch1Only.joinToString("\n"))
+            println()
+        }
+
+        if (commitsInBranch1Only.isNotEmpty()) {
+            println("Commits in '$branch1' but not '$branch2':")
+            commitsInBranch1Only.forEach(::printCommitInfoLine)
+        } else {
+            println("✅ No unique commits found in '$branch1'.")
+        }
+
+        println()
+        println(horizontalLine)
+
+        val branch1Subjects = commits1.map { it.subject }.toSet()
+        val commitsInBranch2Only = commits2.filter { (_, subject) -> subject !in branch1Subjects }
+
+        if (verbose) {
```

Comment:
Nit: How about using a logger that supports multiple levels and allows setting the min log level?